hobbies.txt

```diff
@@ -1,2 +1,2 @@
 My hobbies are singing, playing football and cricket.
-I have started exercising regularly.
\ No newline at end of file
+I have started exercising regularly. I am an avid cricket fan.
```

Comment:
Like, here can you separate them in two lines.

Reply:
I have updated it. You can mark it as resolved.